integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/standardpartition/StandardPartitionTableQueryTestCase.scala

```diff
@@ -243,11 +242,12 @@ class StandardPartitionTableQueryTestCase extends QueryTest with BeforeAndAfterA
 
   }
 
-test("Creation of partition table should fail if the colname in table schema and partition column is same even if both are case sensitive"){
-  intercept[Exception]{
-    sql("CREATE TABLE uniqdata_char2(name char,id int) partitioned by (NAME char)stored by 'carbondata' ")
+  test("Creation of partition table should fail if the colname in table schema and partition column is same even if both are case sensitive") {
+    //TODO: error message is improper, it is different between spark2.1 and spark2.2
+    intercept[AnalysisException] {
+      sql("CREATE TABLE uniqdata_char2(name char,id int) partitioned by (NAME char)stored by 'carbondata' ")
```

Comment:
i fixed this test case with proper exception checking logic for both spark version and comments in pr #2034

Reply:
ok